docs/guide/building.md

```diff
@@ -0,0 +1,22 @@
+# Building
+
+> Part of Open Web Component Recommendation [open-wc](https://github.com/open-wc/open-wc/)
+
+[![CircleCI](https://circleci.com/gh/open-wc/open-wc.svg?style=shield)](https://circleci.com/gh/open-wc/open-wc)
+[![BrowserStack Status](https://www.browserstack.com/automate/badge.svg?badge_key=M2UrSFVRang2OWNuZXlWSlhVc3FUVlJtTDkxMnp6eGFDb2pNakl4bGxnbz0tLUE5RjhCU0NUT1ZWa0NuQ3MySFFWWnc9PQ==--86f7fac07cdbd01dd2b26ae84dc6c8ca49e45b50)](https://www.browserstack.com/automate/public-build/M2UrSFVRang2OWNuZXlWSlhVc3FUVlJtTDkxMnp6eGFDb2pNakl4bGxnbz0tLUE5RjhCU0NUT1ZWa0NuQ3MySFFWWnc9PQ==--86f7fac07cdbd01dd2b26ae84dc6c8ca49e45b50)
+[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
+
+Building is a necessity for application performance and supporting older browsers. Roughly speaking build tools usually take care of the following concerns:
+1. Code optimization (minification, dead code removal)
+2. Bundling
+3. Module resolution
+4. Transforming/polyfilling for older browsers
+5. Supporting non-standard code patterns
+
+We think it's best to avoid nr. 5 as non-standard code will lock your source code to a specific build setup which can be hard to escape in the future. Javascript is evolving fast, your code can quickly become out of sync with new development.
+
+## Webpack
+Webpack is a popular and flexible build tool, with a growing ecosystem of plugins. We recommend as a general purpose tool to get started. See [building-webpack](https://open-wc.org/recommendations/building-webpack.html) to get started.
```

Comment:
```suggestion
Webpack is a popular and flexible build tool, with a growing ecosystem of plugins. We recommend it as a general purpose tool to get started. See [building-webpack](https://open-wc.org/building/building-webpack.html) to get started.
```

Reply:
updated suggestion to relative `/building/building-webpack` so it also links correctly on the preview pages